assets/src/main/java/bisq/asset/coins/Doichain.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+
+public class Doichain extends Coin {
+
+    public Doichain() {
+        super("Doichain", "DOI", new DefaultAddressValidator());
```

Comment:
Do not use deprecated validator.

Reply:
we are strongly based on Namecoin, so I used Namecoin as an example. This probably means this PR gets rejected. So I'll look again to find an up-to-date validator.